features/step_definitions/gegeven-stepdefs-verblijfplaats.js

```diff
@@ -66,3 +68,40 @@ Given(/^er zijn (\d*) personen ingeschreven op adres '(.*)' met de volgende gege
         createPersoonMetWoonadres(this.context, burgerservicenummer, adresId, dataTable);
     }
 });
+
+Given(/^met datum aanvang adreshouding (\d*) in gemeente '(\d*)' zijn de volgende personen ingeschreven op adres '(.*)'$/, function (datumAanvang, gemeenteCode, adresId, dataTable) {
+    // De dataTable moet minimaal een kolom 'burgerservicenummer (01.20)' hebben
```

Comment:
Hier word ik niet blij van. Dit betekent dat gebruikers van deze stap moeten weten dat burgerservicenummers in een kolom moet worden opgegeven. Als we deze stappen niet willen rewriten/vervangen, dan moet er wel worden gecheckt dat de burgerservicenummer kolom voorkomt. Anders leidt dit tot een technische foutmelding die niet-techneuten niet zullen begrijpen

Reply:
mee eens. ben daar kort door de bocht gegaan want wist niet wat ik in de code moet doen in geval dat een fout optreedt

nu komt er geen foutmelding wanneer ik geen burgerservicenummer opneem, er wordt gewoon een persoon opgevoerd zonder burgerservicenummer. is dat beter of erger dan een onduidelijke technische fout....

@MelvLee ik heb nu toegevoegd:
```
        if (!dataRow.hasOwnProperty('burgerservicenummer (01.20)') ) {
            throw new Error("In de data tabel ontbreekt kolom 'burgerservicenummer (01.20)'");
        }
```

is dat ook hoe jij dat zou doen?